### PR TITLE
Hotfix tests

### PR DESCRIPTION
**Pull request by ethernal12**
Adding hotFix for feature tests, the hotFix changes include:

- added fetchSchema.js file, which is used for importing schemas into the tests
- added msg1 initialization in appointments / PUT test
- removed console logging from appointments tests

Please for code review. @urosjarc

### CODE REVIEW

File: test/fetch_schema.js [hunk 1]
```diff
@@ -0,0 +1,2036 @@
+
+function add(feature_test, request_method) {
```
Comment (urosjarc) [inline test/fetch_schema.js]: @ethernal12 : Daj sheme v neko posebno datoteko (schemas) in notri potem daš za vsak route posebaj file... schema_appointments, schema_messages etc...

**Review by urosjarc — CHANGES_REQUESTED** [review]

File: test/fetch_schema.js (hunk 1 above)
Comment (urosjarc) [reply test/fetch_schema.js]: Schem je kar veliko zato se jih splača dati v vsak svoje file.

Comment (ethernal12) [reply test/fetch_schema.js]: Misliš folder? Tako da ustvarim folder schemas in potem notri priložim za vsak route posebaj .js file ?

Comment (urosjarc) [reply test/fetch_schema.js]: @ethernal12 Ja da v folder daš za vsak route svojo shemo da so sheme popolnoma ločene od test fileov... Fetch_schema daj v en direktorij in jo razdeli na več fileov (vsak route posebaj)

Comment (ethernal12) [reply test/fetch_schema.js]: Ok, bom popravil.

**Comment by ethernal12** [comment]
Variable teste še dodam, ko bo hotFix pull request zaključen.

**Pull request merged**